python/pipeline/pupil.py

```diff
@@ -1,35 +1,48 @@
 # Disable DLC GUI first, then import deeplabcut
 import os
 os.environ["DLClight"] = "True"
+
 import deeplabcut as dlc
-from deeplabcut.utils import auxiliaryfunctions
 from .utils import DLC_tools
-
-from .exceptions import PipelineException
-from . import experiment, notify, shared
-from .utils import h5
-from . import config
-from .utils.eye_tracking import PupilTracker, ManualTracker
-from .utils import eye_tracking
-from .utils.decorators import gitlog
-
+from deeplabcut.utils import auxiliaryfunctions
+from itertools import count
+from scipy.misc import imresize
+from tqdm import tqdm
+import shutil
+import cv2
+import numpy as np
+import pandas as pd
+import json
+from commons import lab
 import datajoint as dj
-from datajoint.autopopulate import AutoPopulate
 from datajoint.jobs import key_hash
+from datajoint.autopopulate import AutoPopulate
 
-from commons import lab
-import json
-import pandas as pd
-import numpy as np
-import cv2
-import shutil
-from tqdm import tqdm
+from .utils.decorators import gitlog
+from .utils import eye_tracking, h5, filter_config
+from .utils.eye_tracking import PupilTracker, ManualTracker
+from . import config
+from . import experiment, notify, shared
+from .exceptions import PipelineException
+
+from IPython import display
+import pylab as pl
+import matplotlib.pyplot as plt
+from matplotlib.patches import Ellipse
+import time
+import datetime
 
-from scipy.misc import imresize
-from itertools import count
 
 schema = dj.schema('pipeline_eye', locals())
 
+if os.name == 'nt':
+   dj.config['external-storage'] = dict(protocol='file',
```

Comment:
external-storage is a bit too generic, maybe do external-pupil

Reply:
updated